### PR TITLE
Refactor sign off, according to the new logic

### PR DESCRIPTION
**Pull request by thecalcc**
TGA-59

### CODE REVIEW

File: client/extensions/tga-sign-off/src/components/fields/editor.tsx [hunk 1]
```diff
@@ -48,13 +33,11 @@ export class UserSignOffField extends React.Component<IEditorProps, IState> {
...
             .then((response) => {
                 if (response) {
-                    this.props.setValue({
-                        consent_publish: false,
-                        consent_disclosure: false,
-                        user_id: null,
-                        funding_source: null,
-                        affiliation: null,
-                        sign_date: null,
+                    superdesk.entities.article.patch(this.props.item, {
```
Comment (MarkLark86) [inline client/extensions/tga-sign-off/src/components/fields/editor.tsx:37]: Why doesn't `this.props.setValue` work here anymore? Just wondering why this change, and why client-core needed 360info (aka tga) specific code

**Review by tomaskikutis — APPROVED** [review]
